package.json

```diff
@@ -61,5 +61,8 @@
   "scripts": {
     "test": "standard && tape test/*-test.js | faucet"
   },
+  "engines": {
+    "node": ">=6.0.0"
```

Comment:
not node 5?

Reply:
Afaik destructuring doesn't work with node 5

Reply:
We should rewrite the `[ callback, promise ] = promisify()` logic in this case

Reply:
I don't feel strongly about it. If someone needs node 5 back they can do a PR.

But it does mean we're no longer supporting node versions until their end of life.

Reply:
Then I think we should put back node 5 on travis and fix the code to support it.

Added it back here to test it out https://travis-ci.org/ralphtheninja/levelup/jobs/271278902

Reply:
We don't really need to use destructuring here, might as well just return an object with `.callback` and `.promise` set.